include/seqan3/utility/type_pack/all.hpp

```diff
@@ -0,0 +1,27 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ * \brief Provides seqan3::type_list and associated metaprogramming utilities.
+ */
+
+#include <seqan3/utility/type_pack/traits.hpp>
+
+/*!\defgroup type_pack Type Pack
+ * \brief Provides metaprogramming utilities for working on type packs.
+ * \ingroup utility
+ *
+ * \details
+ *
+ * All traits on a type pack are defined in the header `<seqan3/utility/type_pack/traits.hpp>`.
```

Comment:
Hm I see why you named the folder "type_pack" to be in line with "type_list" but in the documentation I would prefer to call it parameter pack because type pack is not really the name for it?
Maybe also include a link to the parameter pack documentation of parameter packs: https://en.cppreference.com/w/cpp/language/parameter_pack